src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallerFactory.cs

```diff
@@ -74,7 +75,7 @@ public static InstallType GetWorkloadInstallType(SdkFeatureBand sdkFeatureBand,
             return InstallType.FileBased;
         }
 
-        private static bool CanWriteToDotnetRoot(string dotnetDir = null)
+        private static bool CanWriteToDotnetRoot(string dotnetDir = null) // TODO
```

Comment:
This TODO is already handled by the `elevationRequired && !WorkloadInstall.IsUserLocal` check. I need to remove the comment.